version.go

```diff
@@ -15,5 +15,5 @@
 package livekitcli
 
 const (
-	Version = "1.2.8"
+	Version = "1.2.9"
```

Comment:
note, this change is after 1.2.9 release, might consider rolling out a new release.